mozilla_schema_generator/common_ping.py

```diff
@@ -59,6 +63,14 @@ def _update_env(self, schema):
         schema.set_schema_elem(active_addons + ("foreignInstall",), integer)
         schema.set_schema_elem(active_addons + ("version",), string)
         schema.set_schema_elem(active_addons + ("userDisabled",), integer)
+        schema.set_schema_elem(
+            prepend_properties(("environment", "addons", "activeGMPlugins"))
+            + ("additionalProperties", "properties", "applyBackgroundUpdates"),
+            with_description(
+                boolean,
```

Comment:
So far, we support coercion of booleans to integers, but not of integers to booleans. We could change this to an integer field, which would potentially preserve more information if we get strange values that aren't 0 or 1. Otherwise, we'll need to add a case in the live transformation logic to handle casting ints to boolean and capturing such events in a metric.

Reply:
See https://github.com/mozilla/gcp-ingestion/blob/dc628b2ee5cd9300af3d3661ad3727f402afce53/ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/transform/PubsubMessageToObjectNode.java#L554-L564

Reply:
Reading over my older PR, it looks like we should be casting to an integer anyways. The integer can take [on values outside of [0, 1].](https://sql.telemetry.mozilla.org/queries/70455/source)